response_operations_ui/common/respondent_utils.py

```diff
@@ -0,0 +1,34 @@
+def status_enum_to_string(status):
+    dict = {
+        'ACTIVE': 'Active',
+        'CREATED': 'Created',
+        'SUSPENDED': 'Suspended',
+        'LOCKED': 'Locked'
+    }
+    return dict.get(status, None)
+
+
+def status_enum_to_class(status):
+    dict = {
+        'ACTIVE': 'status--success',
+        'CREATED': 'status--info',
+        'SUSPENDED': 'status--warning',
+        'LOCKED': 'status--dead'
+    }
+    return dict.get(status, None)
```

Comment:
2 minor things, the variable name dict should probably be something else; whilst it's a keyword, there are other uses for that word that have semantic meaning so I try to steer away from it where possible.
Second thing, no need to define None as the default as that's what it defaults too already.

Reply:
This comment applies to the other one too

Reply:
Changed to `enum_lookup`.